clients/common/src/api/get-practitioners/index.ts

```diff
@@ -27,7 +29,11 @@ export const parsePractitioners = ({
     })
   }
 
-  onTimeslotLoad()
+  if (!initialized) {
+    onLoad()
```

Comment:
I think I understand why we need to move the `onLoad` to the embed. But why do we invoke that callback here in the `parsePractioners` function?

Is it just because this is the most common codepath for both the appointments and scheduler?

I think that's it because the original author of this library calls the `setLoading` setter here too

Reply:
> I think that's it because the original author of this library calls the setLoading setter here too

That is correct.

I need to verify that the appointments embed works... i'm going to WIP this until tomorrow

Reply:
Thank you for calling out the impact on the appointments...